assets/src/dashboard/app/views/editorSettings/publisherLogo/popoverLogoContextMenu.js

```diff
@@ -50,12 +52,16 @@ function PopoverLogoContextMenu({
   );
 
   const handleFocusOut = useCallback(() => {
-    if (contextMenuId.value === activePublisherLogo) {
+    if (contextMenuId.value === publisherLogo.id) {
       onMoreButtonSelected(-1);
     }
-  }, [activePublisherLogo, contextMenuId.value, onMoreButtonSelected]);
+  }, [publisherLogo.id, contextMenuId.value, onMoreButtonSelected]);
 
-  useFocusOut(popoverMenuContainerRef, handleFocusOut, [contextMenuId]);
+  useFocusOut(
+    popoverMenuContainerRef,
+    contextMenuId.value === publisherLogo.id ? handleFocusOut : noop,
```

Comment:
**Part 2**
After we fixed the bug in part 1, we found a new bug 😕.

After changing that one line, the focus out event handler was firing for all publisher logo ids even if the publisher logo didn't have focus in the first place. This happened no matter where you clicked on the page.
<image src="https://user-images.githubusercontent.com/22185279/109048475-58ae7e00-7694-11eb-8782-37e419acbe8f.gif" width="600px" />

I'm still unsure as to why that happened. However, replacing the callback function with a no-op function fixed this. Even if it calls the function when it's not supposed to, the function does not do anything now.

Reply:
This made the browser behave correctly, however the tests were now failing. Continued in **Part 3**